metrics/ci_integrations/lib/source/buildkite/adapter/buildkite_source_client_adapter.dart

```diff
@@ -42,6 +43,9 @@ class BuildkiteSourceClientAdapter implements SourceClient {
   ) async {
     ArgumentError.checkNotNull(build, 'build');
     final latestBuildNumber = build.buildNumber;
+    Logger.logInfo(
+      'BuildkiteSourceClientAdapter: Fetch builds after build #$latestBuildNumber...',
```

Comment:
Would it be great to add a similar log to the `fetchBuilds` method? Please, apply this suggestion to all the reset source client adapters.